simpleui-server/src/props-file-reader.ts

```diff
@@ -221,10 +249,17 @@ export class PropsFileReader {
 
                 props['macro'][macroIndex]['index'] = macroIndex.toString();
                 props['macro'][macroIndex]['id'] = `macro-${macroIndex}`;
-                props['macro'][macroIndex]['token'] = macroParts[1];
-                props['macro'][macroIndex]['source'] = macroParts[4];
-                props['macro'][macroIndex]['property'] = macroParts[2];
-                PropsFileReader.processMacroPart(props, macroIndex);
+                props['macro'][macroIndex]['token'] = token;
+                props['macro'][macroIndex]['source'] = source;
+                props['macro'][macroIndex]['property'] = property;
+
+                // override dbname with given value
+                if (PropsFileReader.cmdVars.DbName !== "" && token.includes("_MYSQL_DB")) {
+                    Logger.log(LogLevel.INFO, `Overriding macro ${token} with ${PropsFileReader.cmdVars.DbName}`);
```

Comment:
Make it obvious why the override happened:

`Logger.log(LogLevel.INFO, ``Overriding macro ${token} with the value of cmd line arg "--DbName  ${PropsFileReader.cmdVars.DbName}"``);`

Reply:
fixed in my last commit